src/doc/src/reference/unstable.md

```diff
@@ -1271,6 +1272,32 @@ Differences between `cargo run --manifest-path <path>` and `cargo <path>`
 
 ### Documentation Updates
 
+## Edition 2024
+* Tracking Issue: (none created yet)
```

Comment:
No tracking issue has been created yet, so this will have to be updated to include a link when the [RFC](https://github.com/rust-lang/rfcs/pull/3501) gets merged and a tracking issue gets created. 

If there is a tracking issue for `cargo` and `rust`, both links should probably be included.